app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/model/pickup-locations.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define(['ko'], function(ko) {
```

Comment:
This file is also redundant. It always just return empty observable collection. That's an additional layer of abstraction that I think will never be changed.